receiver/carbonreceiver/README.md

```diff
@@ -1,11 +1,15 @@
 # Carbon Receiver
 
+| Status                   |            |
+| ------------------------ |------------|
+| Stability                | [alpha]    |
```

Comment:
Given that it has being out for some time and got some usage some time ago (during a migration, not so much right now for me). I would rather go with `stable` since there isn't any planned work for this receiver.